en/orm/entities.rst

```diff
@@ -313,7 +313,7 @@ method::
     on.
 
 When using the ``newEntity()`` and ``patchEntity()`` methods in the ``Table``
-objects you also have control over the mass assignment protection. Please refer
+objects you can customize mass assignment protection by yourself. Please refer
```

Comment:
I can change protection with "Changing Accessible Fields" 
http://book.cakephp.org/3.0/en/orm/saving-data.html#changing-accessible-fields
If changing protection with it  means "without by myself" , your opinion is right..
How do you think?